### PR TITLE
add gcp instructions

### PR DESCRIPTION
**Pull request by allaway**
@jaybee84 - If you have a chance, could you review and follow the instructions for using these docker images on google cloud platform (can just use a [free trial account with your sage login](https://console.cloud.google.com/freetrial). Hopefully you won't encounter the same `docker login` issues on GCP. :)

### CODE REVIEW

**Comment by jaybee84** [comment]
@allaway : tried out the GCP instructions... worked well to get the GC instance going... but still ran into HTTP 503 error when using docker login docker.synapse.org . Will revisit my docker setup to troubleshoot this problem ..

**Comment by allaway** [comment]
that is super weird. It must have something to do with credentials or your synapse account status, since this is a totally independent install of Docker.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
  
-We believe that open science, collaboration, and talented people are three main ingredients we need to bake a cure for NF. The event plans to attract experts in AI, Genomics, Bioinformatics, Computer Science, and Life Science to glean new insights into these rare diseases.  Gathering these experts to find new associations, new perspectives, and revolutionizing the way we think about NF will create the right turf where great discoveries could happen! 
+We believe that open science, collaboration, and talented people are three main ingredients we need to bake a cure for NF. The event plans to attract experts in AI, genomics, bioinformatics, computer science, and life science to glean new insights into these rare diseases.  Gathering these experts to find new associations, new perspectives, and revolutionizing the way we think about NF will create the right turf where great discoveries could happen!
```
Comment (sgosline) [inline README.md]: 'right turf' sounds a bit colloquial, maybe just 'help make great discovers happen'?

**Comment by allaway** [comment]
will do! this is copied and pasted from the event one-pager, so the
language is more advertise-y than necessary, probably

**Pull request merged**